packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -1036,6 +1083,19 @@ class ScrollableBase extends Component {
 								null
 							}
 						</div>
+						<Button
+							className={classNames(
+								css.scrollToTopButton,
+								this.state.shouldShowScrollToTopButton ? css.showScrollToTopButton : null,
+								isHorizontalScrollbarVisible ? css.horizontalScrollbar : null,
+								isVerticalScrollbarVisible ? css.verticalScrollbar : null,
+								rtl ? css.rtlScrollToTopButton : null
+							)}
+							icon="arrowlargeup"
```

Comment:
Could choose alternate icons for horizontal and horizontal + rtl too. We'll sort that out when the final UX lands.